conan/internal/internal_tools.py

```diff
@@ -4,7 +4,7 @@
 
 
 def is_universal_arch(settings_value, valid_definitions):
-    if settings_value is None:
+    if settings_value is None or valid_definitions is None or universal_arch_separator not in settings_value:
```

Comment:
Interesting the new condition ``universal_arch_separator not in settings_value``, I am adding a check in the unit test to cover it too